HW/Oleksandr Liashenko/HW07_Task3.py

```diff
@@ -0,0 +1,7 @@
+word = input("Enter word: ")
+result = dict()
+for i in word:
+    dictionary = dict.fromkeys(i, word.count(i))
+    result.update(dictionary)
+
```

Comment:
in this case we dont need result dict

Reply:
According to the task, the expected output is: {"h":1, "e":1, "l":2, "o":1}. Is there another way to get it?

Reply:
```
word = input("Enter word: ")
for i in word:
    dictionary = dict.fromkeys(i, word.count(i))
```
and here - dictionary already answer